client/src/components/Onboarding/OnboardingAdmin.jsx

```diff
@@ -233,11 +234,24 @@ OnboardingAdminList.propTypes = {
 const OnboardingAdmin = () => {
   const dispatch = useDispatch();
   const [eventLogModalParams, setEventLogModalParams] = useState(null);
+  const [showIncompleteOnly, setShowIncompleteOnly] = useState(false);
+
+  const toggleShowIncomplete = () => {
+    setShowIncompleteOnly((prev) => !prev);
+  };
 
   const { users, offset, limit, total, query, loading, error } = useSelector(
     (state) => state.onboarding.admin
   );
 
+  const filteredUsers = users.filter((user) => {
+    if (showIncompleteOnly) {
+      return !user.setupComplete;
+    } else {
+      return true;
+    }
+  });
```

Comment:
nit: could make it a bit more concise like this:```suggestion
  users.filter((user) => (showIncompleteOnly ? !user.setupComplete : true));
```